trade-finance-manager-ui/templates/case/amendments/amendment-facility-end-date.njk

```diff
@@ -23,61 +23,66 @@
   }) }}
  {% endif %}
 
-<form method="POST" autocomplete="off" class="govuk-!-margin-top-4">
+<form method="POST" autocomplete="off">
   <input type="hidden" name="_csrf" value="{{ csrfToken }}">
 
-  {{ govukDateInput({
-    id: "facilityEndDate",
-    fieldset: {
-        legend: {
-          text: "Enter the new facility end date",
-          isPageHeading: true,
-          classes: "govuk-fieldset__legend--l govuk-heading-l govuk-!-margin-top-4 govuk-!-margin-bottom-2"
-        }
-      },
-    namePrefix: "facility-end-date",
-    hint: {
-      text: "For example, 31 3 2022",
-      attributes: {
-        "data-cy": "facility-end-date-hint"
-      }
-    },
-    errorMessage: error and {
-      text: error.summary[0].text,
-      attributes: {
-        'data-cy': 'amendment--inline-error'
-      }
-    },
-    items: [
-      {
-        label: "Day",
-        classes: (error and 'facility-end-date-day' in error.fields) and "govuk-input--error govuk-input--width-2" or "govuk-input--width-2",
-        name: "day",
-        value: facilityEndDateDay,
+  <fieldset class="govuk-fieldset" role="group" aria-describedby="coverEndDate-hint">
+
+    <legend class="govuk-fieldset__legend">
+      <h1 class="govuk-heading-l govuk-!-margin-top-4 govuk-!-margin-bottom-0" data-cy="amendment--facility-end-date-heading">Enter the new facility end date</h1>
+    </legend>
+
+    <dl class="govuk-body govuk-!-margin-bottom-4" data-cy="amendment--current-facility-end-date">
+      <dt class="govuk-body govuk-!-margin-bottom-0 govuk-!-font-weight-bold">Current facility end date</dt>
```

Comment:
Looks like dramatic change, but just a small restructure in this file to allow to add in this the 'current facility end date' section